demos/embeddings/stellargraph-keras-node2vec.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>Maybe a little bit more explanation for the loss function would be helpful for guiding users who are not as familiar with Node2Vec. Something like: "Our generator will produce batches of positive and negative context pairs as inputs to the model, so we use binary crossentropy much like a regular binary classification task."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Thanks for your suggestion. I will add the explanation.

Reply:
Thanks! I will add the explanation.